cmake/platform/Linux.cmake

```diff
@@ -3,5 +3,18 @@
 #
 # Linux platform file for standard linux targets. Merely defers to [./Linux-common.cmake](Linux-common.cmake).
 ####
+
+# Set platform default for stubbed drivers
+if (NOT DEFINED FPRIME_USE_STUBBED_DRIVERS)
+   set(FPRIME_USE_STUBBED_DRIVERS ON)
```

Comment:
Do we really want to default to stubbed drivers on linux? I think all of our linux drivers should compile on standard linux systems. I'd be kinda surprised if I compiled on linux and found the `LinuxGPIODriver` was silently stubbed out.

Reply:
I'm a little worried this will present a poor experience for users compiling on their embedded systems, which I think is not an unlikely scenario for systems like the raspberry pi and beaglebone. They'll compile, silently used the stubbed modules, and see the the reference app start but "not work" and actually toggle GPIOs. Personally, I have no concerns compiling the real drivers for desktop systems. Worst case scenario why they try and run the app they'll get permission denied trying to open the hardware, but realistically you shouldn't be trying to actually run most F' deployments on desktop systems. 

It's not a huge deal either way though and if you prefer to leave the drivers stubbed that's okay, we should probably just make sure to explicitly document the configuration option.